Makefile

```
@@ -8,23 +8,23 @@ format: # reformat source python files
 	black src/licensemanager2 src/conftest.py jawthorizer
 
 
-PIP_INSTALL_FOR_ZIP		:= pip install -q --target _lambda_tmp --upgrade wheel pip
-
-
 function.zip:
 	rm -rf $@ _tmp_venv
-	python3 -m venv _tmp_venv
-	. ./_tmp_venv/bin/activate && pip install --upgrade wheel pip \
-	    && pip install .
+	python3 -m virtualenv _tmp_venv
+	. ./_tmp_venv/bin/activate \
+            && pip install --upgrade wheel pip \
+            && pip install .
```

```suggestion
	. ./_tmp_venv/bin/activate 
	pip install --upgrade wheel pip
	pip install .
```

---

This can't be done like this. To keep the venv context in make commands you have to chain the commands.

---

Oh, did not know that.

There is a recommendation to always start continuation lines with the same indentation level, like this:

```suggestion
	. ./_tmp_venv/bin/activate; \
	pip install --upgrade wheel pip; \
	pip install .
```

I think it looks cleaner, but YMMV.